src-electron/db/query-config.js

```diff
@@ -161,6 +165,10 @@ async function insertClusterDefaults(db, endpointTypeId, packageId, cluster) {
   return Promise.all(promises)
 }
 
+async function queryMetaFile(db) {
+  return dbApi.dbAll(db, 'SELECT PATH FROM PACKAGE WHERE PACKAGE_ID = 31')
```

Comment:
Where is the "31" magic constant coming from?

Reply:
it needs to be corrected to dynamically find the package_id rather than this temporary work around

Reply:
fixed this with a query